frontend/packages/dev-console/src/components/monitoring/dashboard/MonitoringDashboardCountBlock.tsx

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react';
+import { Bullseye } from '@patternfly/react-core';
+
+const MonitoringDasboardCountBlock = () => {
+  return (
+    <div className="graph-wrapper">
```

Comment:
nit: can we have class name as `odc-filename-*`

Reply:
I am reusing this class from ```_graph.scss```.